README.md

```diff
@@ -131,6 +131,8 @@ const styles = StyleSheet.create({
 | onDismiss                | `func`    | Callback for when alert is dismissed           | -       |
 | customView               | `object`  | Custom view to render inside alert             | null    |
 | modalProps               | `object`  | Additional props to pass for Modal             | -       |
+| confirmButtonTestID      | `string`  | Confirm button testID                          | awesome-alert-confirm-btn|
+| cancelButtonTestID       | `string`  | Confirm button testID                          | awesome-alert-confirm-btn|
```

Comment:
hey man, can you check the text & default value for the cancel button, seems like a minor copy/paste issue.

Reply:
My bad.  Will amend now.